flashcard.rb

```diff
@@ -0,0 +1,45 @@
+require './lib/helper.rb'
+
+class FlashcardRunner
+  def initialize(round, deck, cards)
+    @round = round
+    @deck = deck
+    @cards = cards
+  end
```

Comment:
This could and should be instantiated only with `round` because round already knows what `deck` and `cards` are. `deck` is only ever being called directly on line 30 which could easily be revised to `@round.deck.cards.size`